### PR TITLE
New api docs suggestions

### PR DESCRIPTION
**Pull request by Wauplin**
- apply suggestion from https://github.com/huggingface/hub-docs/pull/1379#discussion_r1729251902 (see https://github.com/huggingface/hub-docs/commit/cf3dc4ee6a13a56ca4120f3dcaf4e57c91619e37)
- reorder toctree to put "getting started" at the beginning (https://github.com/huggingface/hub-docs/commit/85f7a5d6df2c0f102750279917342f15835cc71e)
- some wording (https://github.com/huggingface/hub-docs/commit/412a026cfcc86f7d7a3b657db696b5e88a3fb64d)

### CODE REVIEW

File: docs/api-inference/parameters.md [hunk 1]
```diff
@@ -21,25 +21,25 @@ To do this, you can add `x-use-cache:false` to the request headers. For example
...
 <curl>
-```bash
+```diff
 curl https://api-inference.huggingface.co/models/MODEL_ID \
     -X POST \
     -d '{"inputs": "Can you please let us know more details about your "}' \
     -H "Authorization: Bearer hf_***" \
     -H "Content-Type: application/json" \
-    -H "x-use-cache: false"
++   -H "x-use-cache: false"
```
Comment (osanseviero) [inline docs/api-inference/parameters.md:30]: Does this work well? I tried in local deploy and this didn't render well

File: docs/api-inference/parameters.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ async function query(data) {
             headers: {
                 Authorization: `Bearer hf_***`,
                 "Content-Type": "application/json",
-                "x-wait-for-model": "true"
++               "x-wait-for-model": "true"
```
Comment (osanseviero) [inline docs/api-inference/parameters.md:135]: Make sure to make type a `diff`

Comment (Wauplin) [reply docs/api-inference/parameters.md:135]: https://github.com/huggingface/hub-docs/pull/1397/commits/8186c69306b90a869eff1c0cda5d42d88741e19a

File: docs/api-inference/parameters.md (hunk 1 above)
Comment (Wauplin) [reply docs/api-inference/parameters.md:30]: @osanseviero highlighting doesn't work locally but it works as expected when deployed. I've opened a dummy PR  (https://github.com/huggingface/huggingface_hub/pull/2492) to test it works: https://moon-ci-docs.huggingface.co/docs/huggingface_hub/pr_2492/en/index?code=python.

image

image


image

**Pull request merged**